wdn/templates_5.0/scss/critical.scss

```diff
@@ -17,8 +17,2034 @@
+//         font-size: 1.5em!important;
+//     }
+// }
+// .unl .dcf-txt-h3 {
+//     font-size: 1.5em!important;
+// }
+// @media only screen and (min-width:56.123125em) {
+//     .unl .dcf-txt-h3 {
+//         font-size: 1.7777777769em!important;
+//     }
+// }
+// .unl .dcf-txt-h2 {
+//     font-size: 1.7777777769em!important;
+// }
+// @media only screen and (min-width:56.123125em) {
+//     .unl .dcf-txt-h2 {
+//         font-size: 1.9999999995em!important;
+//     }
+// }
+.unl .dcf-txt-h1 {
+  font-size: 1.9999999995em!important;
+}
+// @media only screen and (min-width:56.123125em) {
+//   .unl .dcf-txt-h1 {
+//     font-size: 2.3703703686em!important;
+//   }
+// }
+// .unl .dcf-txt-xl {
+//     font-size: 2.3703703686em!important;
+// }
+// @media only screen and (min-width:56.123125em) {
+//     .unl .dcf-txt-xl {
+//         font-size: 2.6666666653em!important;
+//     }
+// }
+// .unl .dcf-txt-2xl {
+//     font-size: 2.6666666653em!important;
+// }
+// @media only screen and (min-width:56.123125em) {
+//     .unl .dcf-txt-2xl {
+//         font-size: 3.160493824em!important;
+//     }
+// }
+// .unl .dcf-txt-3xl {
+//     font-size: 3.160493824em!important;
+// }
+// @media only screen and (min-width:56.123125em) {
+//     .unl .dcf-txt-3xl {
+//         font-size: 3.5555555529em!important;
+//     }
+// }
+// .unl .dcf-txt-4xl {
+//     font-size: 3.5555555529em!important;
+// }
+// @media only screen and (min-width:56.123125em) {
+//     .unl .dcf-txt-4xl {
+//         font-size: 4.2139917643em!important;
+//     }
+// }
+// .unl .dcf-txt-5xl {
+//     font-size: 4.2139917643em!important;
+// }
+// @media only screen and (min-width:56.123125em) {
+//     .unl .dcf-txt-5xl {
+//         font-size: 4.740740736em!important;
+//     }
+// }
+// .unl .dcf-txt-6xl {
+//     font-size: 4.740740736em!important;
+// }
+// @media only screen and (min-width:56.123125em) {
+//     .unl .dcf-txt-6xl {
+//         font-size: 5.6186556843em!important;
+//     }
+// }
+// .unl-ls-0 {
+//     letter-spacing: 0!important;
+// }
+.unl-ls-1 {
+  letter-spacing: 0.01em!important;
+}
+.unl-ls-2 {
+  letter-spacing: 0.1em!important;
+}
+@media only screen and (max-width:56.123125em) {
+  .unl .dcf-breadcrumbs {
+    height: 4.2139917643em;
+    overflow: hidden;
+  }
+  .unl .dcf-breadcrumbs ol {
+    padding-bottom: 4.2139917643em;
+    overflow-x: auto;
+    overflow-y: hidden;
+  }
+  .unl .dcf-breadcrumbs li {
+    flex-shrink: 0;
+  }
+  .unl .dcf-header .dcf-institution-title {
+    display: none;
+  }
+  .unl .dcf-logo-lockup {
+    padding-bottom: 1em;
+    padding-top: 1.333333333em;
+  }
+  .unl-site-title a {
+    font-size: 1em;
+  }
+  .unl .dcf-idm-login {
+    flex-direction: column;
+  }
+  .unl .dcf-idm-toggle {
+    display: none;
+  }
+  .unl .dcf-nav-menu {
+    pointer-events: none;
+    visibility: hidden;
+  }
+  .unl .dcf-nav-toggle-group {
+    display: flex;
+    padding-bottom: env(safe-area-inset-bottom);
```

Comment:
The Travis CI is failing due to this line.

Reply:
Travis CI [is](https://css-tricks.com/the-notch-and-css/) [wrong](https://css-tricks.com/css-environment-variables/).